src/getting-started.md

```diff
@@ -131,204 +125,189 @@ are two major downsides:
 Anki solves these problems by allowing you to split the content of your
 cards up into separate pieces of information. You can then tell Anki
 which pieces of information you want on each card, and Anki will take
-care of creating the cards for you and updating them if you make any
-edits in the future.
+care of creating the cards for you.
 
-Imagine we want to study French vocabulary, and we want to include the
-page number on the back of each card. We want our cards to look like
+Imagine we want to study French vocabulary, and we want to include a
+page number on the back of each card. We want the back of our cards to look like
```

Comment:
In the beginning example chem example, users are shown the back of a card which includes both question and answer. It's the answer side here.

Reply:
I see the parallel you're drawing, but here, we're talking about rearranging the fields of information to create both the question and answer sides of the card. After all, this wouldn't be that great of a feature if you could only use it to change things on the _back_.

Reply:
Then it shouldn't be in the same code block because this actually tripped me up. Even then, talking about both templates at the same time is confusing for the reader.
 I also don't think the goal here is to let people know about all the things you can do in Anki, but to give them guidance instead to be able to discover and use it for themselves. Both the templates are mentioned and people understanding one of them is enough.

Reply:
I think an average user is much more concerned with what happens on the _front_ of their cards, than the back. But this clearly denotes a Q side and an A side.